yw_website/yw_website/apps/website/views.py

```diff
@@ -86,7 +86,7 @@ def my_workflows(request):
     workflows = paginator.get_page(page)
     host = request.get_host()
     return render(
-        request, "pages/my_workflows.html", {"workflow_list": workflows, "host": host}
+        request, "pages/my_workflows.html", {"workflow_list": workflows, "host": host, "can_edit_tags":True}
```

Comment:
you'll want to pass in false for `can_edit_tags` in the `home` view

Reply:
if it doesn't exist it is false so we don't need to

Reply:
> You still need to pass the `edit` and `can_edit_tags` value into the included `tagarray.html` for both the `_base_detailed_workflow.html` and `_base_workflow_listview.html` respectively.



> You still need to pass the `edit` and `can_edit_tags` value into the included `tagarray.html` for both the `_base_detailed_workflow.html` and `_base_workflow_listview.html` respectively.

don't need to cast the can_edit_tags name for my_workflows as it uses the same name

Reply:
i like to to be explicit but if you like it this way better thats fine

Reply:
> don't need to cast the can_edit_tags name for my_workflows as it uses the same name

that's sort of hacky, it assumes that this includeable is in a page that passed those values again. 

Not a big fan, again, I think it's better to be explicit because other people need to develop this and being explicit makes it easier for someone unfamiliar to develop

Reply:
it also means that this includeable is using a variable from a higher scope, which in general is bad practice.

Reply:
Yea, it is a nested includable so I'm not sure what the best practice would be